src/e2e/Dockerfile

```diff
@@ -15,5 +15,6 @@ ENV CHROME_BIN /usr/bin/chromium-browser
 WORKDIR /client-tests
 
 COPY . .
+RUN npm set unsafe-perm true
```

Comment:
Why has this been added?

Reply:
There is a postinstall script as part `e2e` that downloads and installs the selenium server jars/binaries globally. This step was throwing an error. My understanding is the script is attempting to set the wrong uid. This tells npm we're running with root privileges so set the uid accordingly. Voila, the problem went away.